README.md

```diff
@@ -1,6 +1,6 @@
 # CrowdStrike Falcon Helm Chart and Helm Operator
 
-[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/falcon-helm)](https://artifacthub.io/packages/search?repo=falcon-helm)
+[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/falcon-helm)](https://artifacthub.io/packages/search?repo=falcon-helm) [![Docker Repository on Quay](https://quay.io/repository/crowdstrike/falcon-helm-operator/status "Docker Repository on Quay")](https://quay.io/repository/crowdstrike/falcon-helm-operator)
```

Comment:
```suggestion
[![Artifact HUB](https://img.shields.io/endpoint?url=https://artifacthub.io/badge/repository/falcon-helm)](https://artifacthub.io/packages/search?repo=falcon-helm) [![Container Repository on Quay](https://quay.io/repository/crowdstrike/falcon-helm-operator/status "Container Repository on Quay")](https://quay.io/repository/crowdstrike/falcon-helm-operator)
```

Reply:
``Docker`` registry or ``container``?

Reply:
Mind holding off on this change? I am going to add another badge with another PR for an OLM bundle, and then, I plan on restructuring the readme, which at that point, I plan on fixing this. Feel free to disagree.

Reply:
No problem!